index.yaml

```diff
@@ -18530,11 +18530,11 @@ entries:
       catalog.cattle.io/release-name: gopaddle
     apiVersion: v2
     appVersion: "5"
-    created: "2024-06-05T00:55:25.01459904Z"
+    created: "2024-06-06T15:15:11.685530761-06:00"
     description: Kubernetes IDE with AI Co-pilot
-    digest: 4f56e39c70605bdeac453f012e472df3652b245a9d9073e4d902f31cd7bae8ff
+    digest: 9791c680e52099b267dfadb74275c7a5de0b866dbfdbadb656ac6c254acd8aae
     home: https://gopaddle.io
-    icon: file://assets/icons/gopaddle.png
+    icon: https://gopaddle-marketing.s3.ap-southeast-2.amazonaws.com/gopaddle.png
```

Comment:
@adamkpickering We discussed the use of external resources for `icon`. This impacts air-gapped environments.

Reply:
Hmm that shouldn't have happened. I will investigate

Reply:
Looks like this was resolved by the automated update that happens overnight.